modules/logger/logger.go

```diff
@@ -7,36 +7,135 @@ import (
 	"os"
 	"runtime"
 	"strings"
+	gotesting "testing"
 	"time"
 
 	"github.com/gruntwork-io/terratest/modules/testing"
 )
 
+var (
+	// DefaultLogf will be used if the caller uses the function Logf, where on the first call to that function, a sane
+	// default logging function will be set. We keep that in a global variable so that we don't need to define it on
+	// every call to Logf.
+	// DefaultLogf is the default logger that is used if no other logger should be provided. It uses the TerratestLogger
+	// to log messages. This can be overwritten to change the logging globally.
```

Comment:
I'm confused by the docs here. Perhaps reword to say something like: `Defaul`t is the default logger that is used for the `Logf` function?

Reply:
there was also the comment still there, removed that and implemented your suggestion.